core/api/transport/session.hpp

```diff
@@ -16,6 +16,11 @@
 #include "api/transport/rpc_io_context.hpp"
 
 namespace kagome::api {
+  enum struct SessionType {
+    kSessionType_Http = 1,
+    kSessionType_Ws,
```

Comment:
```suggestion
    kHttp = 1,
    kWs,
```
Why duplicate the name of a scoped enum in its constants?

Reply:
agree